topo/node/openconfig/openconfig.go

```diff
@@ -31,14 +33,41 @@ import (
 	log "k8s.io/klog/v2"
 )
 
+const (
+	// modelMagna is a string used in the topology to specify that a magna (github.com/openconfig/magna)
+	// ATE instance should be created.
+	modelMagna string = "MAGNA"
+	// modelLemming is a string used in the topology to specify that a lemming (github.com/openconfig/lemming)
+	// device instance should be created.
+	modelLemming string = "LEMMING"
+	// ondatraRoleLabel is a string used to specify whether an entity should be considered a DUT or an ATE
+	// by ONDATRA. OpenConfig is a vendor who supplies both.
+	ondatraRoleLabel string = "ondatra-role"
```

Comment:
these 3 consts can be imported from the node package

Reply:
Done, thanks, I had implemented this before merging `main` so had missed these. Thanks for pointing them out.